framework/src/controller/application.js

```diff
@@ -0,0 +1,255 @@
+const assert = require('assert');
+const Controller = require('./controller');
+const defaults = require('./defaults');
+const version = require('../version');
+const validator = require('./helpers/validator');
+const schema = require('./schema/application');
+const { createLoggerComponent } = require('../components/logger');
+
+const ChainModule = require('../modules/chain');
+
+// Private scope used because private keyword is restricted
+const scope = {
```

Comment:
```suggestion
const private = {
```

Reply:
I tried private before, but eslint was complaining.

Reply:
do we need to put this in nested object?

Reply:
@shuse2 having a nested object increases code readability I think. It helps to differentiate the private variables from global/public variables.